requirements.txt

```diff
@@ -16,11 +16,12 @@ ops.interface_azure @ git+https://github.com/charmed-kubernetes/interface-azure-
 aiohttp == 3.7.4
 cosl == 0.0.7
 jinja2 == 3.1.3
-loadbalancer_interface == 1.2.0
+loadbalancer_interface == 1.2.1
 ops == 2.12.0
 pydantic == 1.10.15
 tenacity == 8.2.3
 # kv and vault-kv
 hvac == 2.2.0
 # auth-webhook service
 gunicorn == 23.0.0
+setuptools == 44.0.0
```

Comment:
Looks like we can drop this -- if we update charmcraft.yaml to have the `charm-binary-python-package` addition

Reply:
Sure, let's use the `charm-binary-python-packages` approach